requirements/pytorch/docs.txt

```diff
@@ -1,6 +1,7 @@
 -r ../docs.txt
 
 ipython[notebook]
+setuptools<=58.0  # workaround for `error in ipython setup command: use_2to3 is invalid.`
```

Comment:
This didn't work to fix https://github.com/Lightning-AI/lightning/actions/runs/3299019493/jobs/5441995066#step:5:774

Any ideas @akihironitta?

```suggestion
```

Reply:
Haven't investigated this error in detail yet, but let me try `<58.0` as a workaround because the breaking change was introduced in 58.0.0.

> Removed support for 2to3 during builds. 
https://setuptools.pypa.io/en/latest/history.html#id393

```suggestion
setuptools<58.0  # workaround for `error in ipython setup command: use_2to3 is invalid.`
```

Reply:
Didn't work. It seems like an issue with `ipython` since it tries to install all versions but cannot find one: https://github.com/Lightning-AI/lightning/actions/runs/3299189002/jobs/5442369638#step:5:547

Reply:
It seems so. I'll look into it

Reply:
For the note, the previously failing job is now successful: https://github.com/Lightning-AI/lightning/actions/runs/3312118396/jobs/5468513948